src/saltext/vmware/modules/storage_policies.py

```diff
@@ -0,0 +1,477 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License: Apache-2.0
+import logging
+import json
+
+import salt.exceptions
+import saltext.vmware.utils.connect as connect
+
+log = logging.getLogger(__name__)
+
+try:
+    from pyVmomi import pbm, VmomiSupport, SoapStubAdapter, vim
+
+    HAS_PYVMOMI = True
+except ImportError:
+    HAS_PYVMOMI = False
+
+
+__virtualname__ = "storage_policy"
+
+
+def __virtual__():
+    if not HAS_PYVMOMI:
+        return False, "Unable to import pyVmomi module."
+    return __virtualname__
+
+def _connect_spbm_(stub, context):
+   hostname = stub.host.split(":")[0]
+   sessionCookie = stub.cookie.split('"')[1]
```

Comment:
**comment (non-blocking)** this may be fine for a follow-up PR but https://docs.python.org/3/library/http.cookies.html#http.cookies.BaseCookie.load exists and, assuming that the cookie is an actual HTTP valid format for the cookie then it's probably more reliable to use Python's https.cookies to load/parse the cookie value(s)